cross/libffi/Makefile

```diff
@@ -14,6 +14,10 @@ LICENSE  = libffi BSD-like
 GNU_CONFIGURE = 1
 PRE_CONFIGURE_TARGET = libffi_pre_configure
 
+# Otherwise the generated pkg-config resources would basically contain invalid information
+# On file locations
+CONFIGURE_ARGS = --prefix=$(INSTALL_PREFIX)
+
```

Comment:
In my opinion, this line is redundant after `GNU_CONFIGURE=1` if you look at configure Makefile targets: https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.configure.mk#L40

Reply:
I'll check what $(TC_CONFIGURE_ARGS) are, just to be sure.

Reply:
Hum that's odd. If I remove this line I get as contents the actual deployed value (e.g. for python3 build `/var/packages/python3/target`).
Here my goal was to have `<path to spksrc>/spk/python3/work-<arch>/var/packages/python3/target`.
Maybe it would be better for me to fix this in the cross/python3 build then ?

Reply:
GNU configure prefix is used to locate binary and share objects at runtime. It cannot be compilation destination directory. If python build chain is not able to locate library, it is required to provide additional options at build so that it finds it.

Reply:
OK I'll remove this and try to fix the issue I had with libffi another way.